src/css.js

```diff
@@ -303,7 +316,8 @@ jQuery.extend( {
 			num = parseFloat( val );
 			return extra === true || isFinite( num ) ? num || 0 : val;
 		}
-		return val;
+
+		return val === undefined ? undefined : jQuery.trim( val );
```

Comment:
Testing in chrome returned the computed values with a preceding space in the string. I will look into cleaning this up.